src/Data/Profunctor/Linear.hs

```diff
@@ -9,6 +9,20 @@
 {-# LANGUAGE TupleSections #-}
 {-# LANGUAGE TypeOperators #-}
 
+-- | This module provides profunctor classes and instances
+--
+-- Please import this module qualified.
+--
+-- This module is heavily connected to and motivated by linear optics.
```

Comment:
I'd prefix this sentence with "some of the definitions on this module ...". Because `Profunctor`s are useful outside of optics to, and I even think it's more common to see them without optics.

But this comment applies more to things like `Wandering`, `Exchange` eg.

Reply:
I'd also argue that those should be defined in Optics modules instead in here, but not super important.